### PR TITLE
feat: Delete ownership grant updates

### PR DESCRIPTION
**Pull request by cinglish**
## Test Plan

* [ ] acceptance tests

* [ ] …

## References

*

### CODE REVIEW

**Comment by cinglish** [comment]
This PR adds an attribute, revert_ownership_to, to both the user_ownership_grant and role_ownership_grant resources to transfer ownership to the provided role instead of defaulting to ACCOUNTADMIN. This fixes an issue we were facing in which we could not run terraform destroy successfully due to DROP ROLE statements failing on snowflake_role resources after their respective snowflake_role_ownership resources were deleted.

File: pkg/resources/role_ownership_grant.go [hunk 1]
```diff
@@ -40,6 +40,16 @@ var roleOwnershipGrantSchema = map[string]*schema.Schema{
 			"REVOKE",
 		}, true),
 	},
+	"revert_ownership_to": {
```
Comment (sfc-gh-swinkler) [inline pkg/resources/role_ownership_grant.go:43]: for consistency perhaps this should be renamed to:
`revert_ownership_to_role_name`
alternatively:
`default_owner_role_name`

**Review by sfc-gh-swinkler — CHANGES_REQUESTED** [review]

**Comment by sfc-gh-swinkler** [comment]
@cinglish can you please run `make docs` again? since the schema changed after you renamed the attribute. thanks in advance!

**Review by sfc-gh-swinkler — APPROVED** [review]

**Comment by sfc-gh-swinkler** [comment]
@cinglish thank you for your contribution!

**Pull request merged**